README.md

```diff
@@ -1,21 +1,18 @@
 You must insert the source code below at the bottom of the body tag to render the Medkudo Book An Appointment.
 
 ```html
+<script src='//sdk.medkumo.loc:85/medkumo.js'></script>
 <script>
-  (function() {
-    var script, entry;
-    script = document.createElement('script');
-    script.async = true;
-    script.src =  '//sdk.medkumo.loc:' + window.location.port + '/medkumo.js';
-    entry = document.getElementsByTagName('script')[0];
-    entry.parentNode.insertBefore(script, entry);
-  })();
-  window.Medkumo_ready = function() {
-    Medkumo.init('<your hospital key>');
-  };
+  function myFunction() {
+    Medkumo.showBookAnAppointment('<hospital_key>', '<doctor_key>')
+  }
 </script>
 ```
-
+After that you can be bind click event to the function.
+Example:
+```html
+<button onclick="myFunction()">Booking</button>
```

Comment:
how about this ?
<button onclick="Medkumo.showBookAnAppointment('<hospital_key>', '<doctor_key>')">Booking</button>